tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java

```diff
@@ -258,7 +260,7 @@ private boolean flushAllChunkGroups() throws IOException {
               chunkGroupFooter.getDataSize(), fileWriter.getPos() - pos));
         }
 
-        fileWriter.endChunkGroup(chunkGroupFooter);
+        fileWriter.endChunkGroup(chunkGroupFooter, version++);
```

Comment:
Suppose we have many TsFiles, and they are generated by different programs.. 
When we put all of them into an IoTDB instance (e.g., using Postback module), maybe we need a version repairing tool .
(There is no need to add the function into this PR, but if so, we need record an issue at least.)

Reply:
It is not a big issue. Stand-alone TsFile does not support deletion. After importing all the TsFiles, any new deletion takes effect on all of them. So the only thing we need to do is to make the version of the deletion greater than those of all existing TsFiles, which is straight-forward.